src/index.js

```diff
@@ -12,16 +12,3 @@ export function addEventListener(target, eventName, listener, options) {
   const normalizedEventOptions = normalizeEventOptions(options);
   return target[EVENT_HANDLERS_KEY].add(eventName, listener, normalizedEventOptions);
 }
-
-export function removeEventListener(eventHandle) {
```

Comment:
this could be left semver-minor by changing `removeEventListener` to *invoke* `eventHandle` if it's a function - that way existing code would continue to work, it'd just be a new feature.